router/router_test.go

```diff
@@ -320,12 +321,14 @@ var _ = Describe("router", func() {
 
 	BeforeEach(func() {
 		conf = config.New()
-		routerUtils.JobRetention = time.Duration(175200) * time.Hour // 20 Years(20*365*24)
+		config.Reset()
+		config.Set("Router.jobRetention", "175200h") // 20 Years(20*365*24)
```

Comment:
shouldn't it be:
```suggestion
		conf.Reset()
		conf.Set("Router.jobRetention", "175200h") // 20 Years(20*365*24)
```

and `conf.Reset()` in `AfterEach`

Reply:
unfortunately this is not possible, needs further refactoring that I am not willing to perform at this stage (global config is used for this specific option)